jobs/io/ndb_io.py

```diff
@@ -74,32 +72,25 @@ def expand(self, pbegin):
 class PutModels(beam.PTransform):
     """Writes NDB models to the datastore."""
 
-    def __init__(self, datastoreio_stub, label=None):
-        """Initializes the PutModels PTransform.
-
-        Args:
-            datastoreio_stub: stub_io.DatastoreioStub. The stub instance
-                responsible for handling datastoreio operations.
-            label: str|None. The label of the PTransform.
-        """
-        super(PutModels, self).__init__(label=label)
-        self.datastoreio = datastoreio_stub
-
-    def expand(self, model_pcoll):
+    def expand(self, entities):
         """Writes the given models to the datastore.
 
+        This overrides expand from parent class.
+
         Args:
-            model_pcoll: PCollection. A PCollection of NDB models.
+            entities: PCollection. A PCollection of NDB models to write
+                to the datastore. Can also contain just one model.
 
         Returns:
-            PCollection. An empty PCollection.
+            PCollection. An empty PCollection. This is needed because all expand
```

Comment:
expand --> expand()

(Otherwise, it takes on its colloquial English meaning, which makes the sentence not make sense.)

Reply:
Done.